### DISCUSSION TITLE
When will new analyzer be ready for production?

### DISCUSSION THREAD

**Discussion by JackyWoo** [ClickHouse/ClickHouse#51650] [Q&A] [answered]
May I ask is there a plan when new analyzer will be ready for production?

**Comment by den-crane** [comment]
Nobody knows, yet. Eventually.

**Reply by JackyWoo** [reply]
Thank you.

**Comment by UnamedRus** [comment, marked as answer]
https://github.com/ClickHouse/ClickHouse/issues/51660#issuecomment-1614427986

**Reply by JackyWoo** [reply]
Thank you.
